src/Admin.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\Gallerya\Admin.
+ */
+
+namespace Netzstrategen\Gallerya;
+
+/**
+ * Administrative back-end functionality.
+ */
+class Admin {
+
+  /**
+   * Plugin backend initialization method.
+   *
+   * @implements admin_init
+   */
+  public static function init() {
+
+    // Adds WooCommerce specific features.
+    if (Plugin::isPluginActive('woocommerce/woocommerce.php')) {
+
+      // Adds custom product data tabs / panels.
+      add_filter('woocommerce_product_data_tabs', __NAMESPACE__ . '\WooCommerce::woocommerce_product_data_tabs');
+      add_action('woocommerce_product_data_panels', __NAMESPACE__ . '\WooCommerce::woocommerce_product_data_panels');
+      add_action('admin_head', __CLASS__ . '::addCustomIconsToTabs');
+    }
+
+  }
+
+  /**
+   * Sets icons for custom product data tabs.
+   *
+   * @implements admin_head
+   */
+  public static function addCustomIconsToTabs() {
+
+    // Adds icon for the 'Videos' tab.
+    echo '<style>
+      #woocommerce-product-data ul.wc-tabs li._' . Plugin::L10N . '_videos_options._' . Plugin::L10N . '_videos_tab a:before{
```

Comment:
This string concatenation results into this selector: `li._gallerya_videos_options._gallerya_video_tab`.
To me looks really long and complex. Is there any other better way of selecting the necessary DOM element?